src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/StartupProjectRegistrar.cs

```diff
@@ -132,7 +133,8 @@ protected override Tasks.Task DisposeCoreAsync(bool initialized)
         {
             foreach (var provider in _launchProviders.Value.Debuggers)
             {
-                if (await provider.Value.CanLaunchAsync(DebugLaunchOptions.DesignTimeExpressionEvaluation)
+                if (string.Equals(DebuggerName, provider.Metadata.DebuggerName, StringComparison.OrdinalIgnoreCase) &&
```

Comment:
I'm not convinced this is the correct solution. You shouldn't know about specific debugger providers - what if launch profiles aren't enabled?